tests/xharness/Jenkins.cs

```diff
@@ -134,7 +151,10 @@ IEnumerable<RunSimulatorTask> CreateRunSimulatorTaskAsync (XBuildTask buildTask)
 				break;
 			case TestPlatform.iOS_Unified:
 				targets = new AppRunnerTarget [] { AppRunnerTarget.Simulator_iOS32, AppRunnerTarget.Simulator_iOS64 };
-				platforms = new TestPlatform [] { TestPlatform.iOS_Unified32, TestPlatform.iOS_Unified64 };
+				if (IncludeiOS32)
```

Comment:
This will prevent the 32-bit tests from showing up. It's better to ignore them instead:

```csharp
bool [] ignored;
...
case TestPlatform.tvOS:
    ignored = new bool [] { false };
    ...
case TestPlatform.watchOS:
    ignored = new bool [] { false };
    ...
case TestPlatform.iOS_Unified:
    ignored = new bool [] { !IncludeiOS32, false };
    ...
...
    runtasks.Add (new RunSimulatorTask (..., Ignored = ignored [i] });
```